spark/src/main/scala/org/tensorflow/spark/datasources/tfrecords/DefaultSource.scala

```diff
@@ -0,0 +1,67 @@
+/**
+ *  Copyright (c) 2016 Intel Corporation 
```

Comment:
Can you change all of these copyright lines to say: `Copyright 2016 The TensorFlow Authors. All Rights Reserved.`?

(Note that regardless, Intel retains copyright. If Intel wants to, they can add themselves to this file list: https://github.com/tensorflow/tensorflow/blob/master/AUTHORS )